src/GraphQL.NewtonsoftJson/SchemaExtensions.cs

```diff
@@ -1,14 +0,0 @@
-using System;
-using System.Threading.Tasks;
-using GraphQL.Types;
-
-namespace GraphQL.NewtonsoftJson
-{
-    public static class SchemaExtensions
-    {
-        public static Task<string> ExecuteAsync(this ISchema schema, Action<ExecutionOptions> configure)
```

Comment:
Was this method deleted or moved? I remember that @joemcbride insisted that this method should remain.

Reply:
Removed. Was being used as a convenience in tests, but after refactoring off, didn't need it and didn't want to make a S.T.Json one. Do we really need it @joemcbride ?

Reply:
See this thread https://github.com/graphql-dotnet/graphql-dotnet/pull/1448#discussion_r354562378

Reply:
No worries. Added it back, with a full comment, and one for S.T.Json as well.